packages/experimental/Drawer/src/DrawerTokens.ts

```diff
@@ -0,0 +1,23 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { DrawerTokens } from './Drawer.types';
+
+export const defaultDrawerTokens: TokenSettings<DrawerTokens, Theme> = (t: Theme) =>
+  ({
+    backdropColor: '#00000080', // to be replace with token in future based on design guidance
+    drawerBackgroundColor: t.colors.neutralBackground2,
+    handleBackgroundColor: t.colors.neutralStroke1,
+    handleWidth: globalTokens.size360,
+    handleHeight: globalTokens.size40,
+    handleCornerRadius: globalTokens.size20,
+    handleAlignment: 'center',
+    handleMarginTop: globalTokens.size80,
+    handleMarginBottom: globalTokens.sizeNone,
+    handleMarginEnd: globalTokens.sizeNone,
+    handleMarginStart: globalTokens.sizeNone,
+    drawerElevation: globalTokens.size100,
+    position: 'left',
```

Comment:
Since position changes the tokens that get applied to the component, they are usually not set as component tokens themselves.

Reply:
Added to useDrawer.